### PR TITLE
Frio: better UX for aside bar

### PR DESCRIPTION
**Pull request by fabrixxm**
In desktop view, aside sticks inside the viewport when user scrolls the page.
In mobile, the aside is shown fixed over the content of the page, removing the need for the user to scroll up to see it.

### CODE REVIEW

**Comment by fabrixxm** [comment]
this is a remake of PR #3372 after my fail at git merge :-(

**Comment by Hypolite** [comment]
Just tested it, I'm slightly put off by the aside taking all the available space on mobile, otherwise it's working as advertised.

Has anyone else tried the mobile behavior to confirm/deny my irk?

**Comment by rabuzarus** [comment]
I have another issue. Try the following: Expand a year in the archive widget. Then you won't be able to scroll down anymore to the bottom of the right aside

**Comment by fabrixxm** [comment]
@Hypolite I've changed the css a little bit in mobile
@rabuzarus should be fixed, if you want to check..

**Comment by fabrixxm** [comment]
untitled

:-)

**Comment by rabuzarus** [comment]
The aside part is working now correct. But on mobile the main network stream section doesn't scroll anymore (if I scroll down it does scroll automatically back to the top  :-/ ).

**Comment by fabrixxm** [comment]
> But on mobile the main network stream section doesn't scroll anymore (if I scroll down it does scroll automatically back to the top :-/ ).

mhfpf... i'ts a problem with Chrome. The desktop version in "mobile" size has same problem.
My solution would be to activate the sticky aside only if the window width is greater than 990px, disabling it totally on mobile view.
This will not be so much "responsive", because is activated or not at page load only...

File: view/theme/frio/js/theme.js [hunk 1]
```diff
@@ -317,17 +317,19 @@ $(document).ready(function(){
 	});
 
 	/*
-	 * Sticky aside on page scroll
+	 * Sticky aside on page scroll, only on desktop
```
Comment (rabuzarus) [inline view/theme/frio/js/theme.js]: Can you please write a note why we have choosen `only on desktop`. It is really helpful after some time, when someone does change somewhere else a behavior and don't find the the source why it isn't working

**Comment by AlfredSK** [comment]
Has someone checked the scrolling on mobile devices? For example on profile page if I open the left side bar I cannot scroll down by wiping unless I do the wiping over the main bar on top of the page.

**Comment by fabrixxm** [comment]
@AlfredSK Just tested on jolla and chrome on android, it works as it should....

**Comment by rabuzarus** [comment]
thx for the good description. I will test it tomorrow

**Comment by rabuzarus** [comment]
sry I don't find any time for testing at the moment. @Hypolite can you test if everything does work as expected? If not, maybe I will find some time until the end of the week

**Comment by Hypolite** [comment]
Just tested on Firefox on iOS, and even after opening the aside, scrolling only seems to affect what's behind the aside.

As a side mention, while I chose the frio theme for my test user, Firefox initially showed me the vier theme on my iPhone. I had to press long on reload, and select "Request the Desktop Site" from the menu that opened.

Not sure why and where it separates the desktop theme from the mobile one, but there's clearly a bug where frio doesn't get to be shown on "mobile" version.

**Comment by rabuzarus** [comment]
OK. I tested it. It does work work correct on Chromium on linux desktop, Android Browser, Firefox for Android.
But @Hypolite seems to have an issue with Firefox on iOS. 
How to resolve this? I don't have iOS and I guess @fabrixxm don't have an iOS system either.

**Comment by Hypolite** [comment]
I would like to add that the tested aside didn't have enough content to be longer than a single screen, not sure if it's relevant, I didn't exactly know what I was testing.

**Comment by fabrixxm** [comment]
yes.. when the scroll space of aside ends, the main page starts to scroll.. if the aside content is shorter than the window, the  page is scrolling.
I need to find a fix to this, because retesting it now is quite ugly as a behavior...

**Comment by fabrixxm** [comment]
The last commit disable page scrolling when the aside is shown.
It adds event listeners for "aside is shown" and "aside is hidden" which add and remove "aside-out" class to `body`element. The css sets `overflow:hidden` to `body.aside-out`, disabling the scroll. The aside container still scroll, but when the user gets to the end of the aside content, the main  content don't start to scroll.. The ux it's much better 👍 :-)

**Comment by rabuzarus** [comment]
Ohhh, I totally forgot to accept that. Sry

**Comment by annando** [comment]
This now went to ```develop``` but not to the RC. Shouldn't it be part of this as well?

**Comment by tobiasd** [comment]
Is it a bug fix? or a new feature?

**Comment by rabuzarus** [comment]
It is a new feature. But on the other side it was submitted 24 days ago

**Comment by tobiasd** [comment]
On Tue, 16 May 2017 11:40:04 -0700
rabuzarus <notifications@github.com> wrote:

> It is a new feature. But on the other side it was submitted 24 days
> ago

Read it's fixing a quirky behaviour in the UX? ;-)

**Comment by tobiasd** [comment]
Hmpf, my git foo it too bad for this it seems ;-)

**Comment by rabuzarus** [comment]
If you want I can create a new PR against the rc branch

**Comment by tobiasd** [comment]
On Wed, 17 May 2017 02:00:14 -0700
rabuzarus <notifications@github.com> wrote:

> If you want I can create a new PR against the rc branch

ok

**Pull request merged**